src/electron/go_vpn_tunnel.ts

```diff
@@ -262,7 +264,14 @@ class GoTun2socks {
           return;
         }
         console.debug('tun2socks started');
-        this.process.onExit = (code?: number, signal?: string) => {
+        this.process.onExit = async (code?: number, signal?: string) => {
```

Comment:
`spawn()`, called by `launch()` will throw on invalid binary: 
![image](https://user-images.githubusercontent.com/113565/121753661-d9c54000-cae0-11eb-9284-905628eb24a0.png)

You should catch the launch call and reject in case of errors. I think that would be cover everything?